mt-st

```diff
@@ -0,0 +1,38 @@
+#mt bash completion by Paweł Marciniak <sunwire<at>gmail.com>
+#
+
+_mt () {
+    local cur prev words cword
+    _init_completion || return
+
+    #possible commands
+    commands="weof wset eof fsf fsfm bsf bsfm fsr bsr fss bss rewind offline rewoffl eject retension eod seod seek tell status erase setblk lock unlock load compression setdensity drvbuffer stwrthreshold stoptions stsetoptions stclearoptions defblksize defdensity defdrvbuffer defcompression stsetcln sttimeout stlongtimeout densities setpartition mkpartition partseek asf stshowopt"
+
+    if [ -x "$(command -v udevadm)" ]; then
+        #list tape devices
+        for d in `udevadm trigger --verbose --dry-run --type=devices --subsystem-match=scsi_tape`; do
```

Comment:
This will re-run udevadm at each tab, not only when prev is `-f`. How about moving this into the `-f` block blow?

Reply:
You are right. It will be a better place for this code.